types/dummy/index.d.ts

```diff
@@ -1 +1,3 @@
-
+declare module 'ember-sinon-qunit' {
+  export default function setupSinon(testEnvironment: Partial<QUnit>): void;
```

Comment:
This won't be a Partial anymore, right?

Reply:
Actually, nvm, I think Partial is the right way to type it because they may in theory supply their own test environment with the same two methods to hook into.

Reply:
In that case, I think that we’d want to  with our own interface, given that we do not know for sure the their test environment is QUnit but we do know the methods they will be using.All that said the addon is called `ember-sinon-qunit` so aren’t we implying they’ll be using QUnit.

```
interface TestEnvironmentInterface {
  testStart: (callback: () => void) => void
  testDone: (callback: () => void) => void
}
```

Reply:
I think it's fine. If we run into a legit use-case, we can always go back and modify the types in a later release.